cmd/root.go

```diff
@@ -51,6 +60,78 @@ func init() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 }
 
+// isArgTuneable returns if the argument provided is allowed to be modified
+func isArgTunable(arg string) bool {
+	return tuneableArgsWhitelist[arg]
+}
+
+// parseTuningFile parses the kernel argument tuning file
+func parseTuningFile(tuningFilePath string) (addArguments []types.TuneArgument, err error) {
+	if tuningFilePath == "" {
+		tuningFilePath = kernelTuningFile
+	}
+	// Return fast if the file does not exist
+	if _, err := os.Stat(tuningFilePath); os.IsNotExist(err) {
+		glog.V(2).Infof("no kernel tuning needed as %s does not exist", tuningFilePath)
+		// This isn't an error. Return out.
+		return addArguments, err
```

Comment:
Did you mean to return `nil` for `err` here? Otherwise, we're percolating up the `ENOENT`, no?

(Also sidenote about golang in general: I find the combination of named return values and explicit `return arg1, arg2, ...` confusing. Seems like we should just drop the names to be clear?)

Reply:
SGTM. Will update.